src/lib.js

```diff
@@ -286,6 +288,16 @@ async function parseBankAccounts($) {
     })
     account.number = number.reference
```

Comment:
Transform `number` in `let` instead of `const`.

```diff
-const number = scrape($('h3.account-number'), {
+let number = scrape($('h3.account-number'), {
```

To simplify your code and avoid duplication code (combo) 😄 
```suggestion
    if (number.reference === '') {
      number = scrape($('div.account-number'), {
        reference: {
          sel: 'strong:first-child'
        }
      })
    }

    account.number = number.reference
    account.vendorId = number.reference
```

Reply:
oh yes I did adding on existing code without to thinking, I update this. Thanks